packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -167,18 +172,27 @@ export function getPopoverDir({
     up = 0;
     down = 0;
   }
-  const spaces = [up, right, down, left];
 
-  // Identify best direction of available spaces
-  const max = Math.max(...spaces);
   // Chose the main direction for the popover based on available spaces & user preference
```

Comment:
nit: should be "Choose"no "Chose" in the comment

Reply:
this code will go away when this one gets merged.>>  ScrollBoundaryContainer: fix positioning within when no available spa… https://github.com/pinterest/gestalt/pull/1421

Reply:
DONE